Modelica/Blocks/Sources.mo

```diff
@@ -118,12 +118,12 @@ variable <strong>y</strong> is both a variable and a connector.
 
   end BooleanExpression;
 
-  block Clock "Generate actual time signal"
+  block ContinuousClock "Generate actual time signal"
     extends Interfaces.SignalSource;
```

Comment:
Not really part of the PR but I wonder what the meaning of "actual" should be here. False friend from German instead of "current"? Or are there also non-actual time signals.

Reply:
Since you asked me (I forgot whether I was the author): "Actual" is not correct. It should be "current". 
The result of this Clock model is, as far as I understand it, a clock signal in the sense of "time", what the watch shows. It is not in the sense of a pulsed clock signal. Therefore, I appreciate the change into ContinuousClock. It is clearer.